Docs/source/running_cpp/parameters.rst

```diff
@@ -1081,51 +1081,109 @@ Numerics and algorithms
     for more information.
 
 * ``psatd.current_correction`` (`0` or `1`; default: `0`)
-    If true, the current correction `(Vay et al, JCP 243, 2013) <https://doi.org/10.1016/j.jcp.2013.03.010>`_
+    If true, a current correction scheme in Fourier space is applied in order to
+    guarantee charge conservation.
+
+    If ``psatd.v_galilean`` is zero, the spectral solver used is the standard PSATD
+    scheme described in
+    `(Vay et al, JCP 243, 2013) <https://doi.org/10.1016/j.jcp.2013.03.010>`_
+    and the current correction reads
 
     .. math::
-       \widetilde{\boldsymbol{J}}^{\,n+1/2}_{\mathrm{correct}} = \widetilde{\boldsymbol{J}}^{\,n+1/2}
-       -\bigg[\boldsymbol{k}\cdot\widetilde{\boldsymbol{J}}^{\,n+1/2}
-       -i\frac{\widetilde{\rho}^{n+1}-\widetilde{\rho}^{n}}{\Delta t}\bigg]
-       \frac{\boldsymbol{k}}{k^2}
+       \widehat{\boldsymbol{J}}^{\,n+1/2}_{\mathrm{correct}} = \widehat{\boldsymbol{J}}^{\,n+1/2}
+       - \bigg(\boldsymbol{k}\cdot\widehat{\boldsymbol{J}}^{\,n+1/2}
+       - i \frac{\widehat{\rho}^{n+1} - \widehat{\rho}^{n}}{\Delta{t}}\bigg) \frac{\boldsymbol{k}}{k^2}
+
+    If ``psatd.v_galilean`` is non-zero, the spectral solver used is the Galilean
```

Comment:
Should we document here that the averaged algorithm combined with the current correction is not currently implemented. 
The implementation will be completed as soon as we move averaging to the `GalileanAlgorithm` class.

Reply:
Yes, thanks for the remainder! Done in 50122f9abfe71030807be9057adc9567867b5dd3 (for the rho-free formulation as well).

Reply:
Also, in 523ae5b1144de199f0130aff3ef1fb71446f8c5a I made the function `CurrentCorrection` pure for all spectral base classes, so that the code raises an error whenever we try to call that function within a PSATD derived class where it is not implemented. Similarly to what we did for the function `VayDeposition` in #1051.